imagetest/test_suites/security/image_security_test.go

```diff
@@ -272,7 +272,7 @@ func verifyAutomaticUpdate(image string) error {
 		return err
 	}
 	switch {
-	case strings.Contains(image, "debian-9"):
+	case strings.Contains(image, "debian"):
```

Comment:
Does this also valid for debian-10?

Reply:
Yup- at least in my test cases it was valid for all debian images (9-11)